python/aitemplate/backend/profiler_runner.py

```diff
@@ -297,14 +298,18 @@ def callback_when_done(fut):
             stdout = None
             stderr = None
             try:
+                stdout, stderr = "", ""
                 stdout, stderr = fut.result()
                 profile_result, err = extract_profile_result(stdout)
                 if err:
-                    _LOGGER.error(
+                    _LOGGER.debug(
                         f"Profiler failure!\nProfiler stdout: {stdout}\nProfiler stderr: {stderr}",
                     )
-                    raise RuntimeError(f"Failed to extract profiler result for {cmds}")
-                process_result_callback(profile_result, self._postprocessing_delegate)
+                    _LOGGER.debug(f"Failed to extract profiler result for {cmds}")
+                else:
```

Comment:
@ipiszy @chenyang78 I'm just highlight some changes, hopefully to make the review a little easier on your side